lnbits/templates/base.html

```diff
@@ -67,10 +67,11 @@
             </q-badge>
             {%endif%} {% if LNBITS_CUSTOM_BADGE is defined %}
             <q-badge
+              v-if="`{{LNBITS_CUSTOM_BADGE}}`.length > 0"
```

Comment:
isn't this what caused the original issue?

Reply:
no. it was a jinja conditional, i think

i tested with a couple of special characters, `'`, `\`, `<` , `;`

all worked